packages/universal/debug/src/setup.ts

```diff
@@ -2,18 +2,11 @@ import { defineDebug } from "@starbeam/reactive";
 
 import debug from "./debug.js";
 
-if (
-  (globalThis.Buffer as BufferConstructor | undefined) === undefined &&
-  typeof require === "function"
-) {
-  try {
-    // this is for CJS only, so require is the only option here
-    // eslint-disable-next-line @typescript-eslint/no-var-requires
-    const buffer = require("buffer") as { Buffer: BufferConstructor };
-    globalThis.Buffer = buffer.Buffer;
-  } catch {
-    // ignore
+if (import.meta.env.DEV) {
```

Comment:
This `import.meta.env.DEV` condition seems sufficient for keeping the debug stuff in development, but make production faster